lib/atlas/graph_from_yaml.rb

```diff
@@ -0,0 +1,60 @@
+module Atlas
+  class GraphFromYaml
+    def self.build(graph_yaml)
+      new(graph_yaml).build_graph
+    end
+
+    def initialize(graph_yaml)
+      @graph_yaml = graph_yaml
+      @graph      = GraphBuilder.build
+    end
+
+    def build_graph
+      @graph.nodes.each do |node|
+        if node_attributes = @graph_yaml.fetch(:nodes)[node.key]
+          atlas_node        = node.get(:model)
+          atlas_node.input  = node_attributes[:in]
+          atlas_node.output = node_attributes[:out]
+
+          node_attributes.except(:in, :out).each_pair do |attr, val|
+            node.set(attr, val)
+          end
+
+          set_slots(atlas_node, node_attributes.slice(:in, :out))
+        end
+      end
+
+      edges.each do |edge|
+        edge.properties.merge!(properties_for_edge(edge))
```

Comment:
But it doesn't really matter because it's changed to `set()` just like the `Slots` which is probably the reason why the `:elastic` wasn't working in the `FullExporter` (see Slack).